pkgs/top-level/python-packages.nix

```diff
@@ -18314,6 +18362,24 @@ in {
       };
     in if isPy3k then py3 else py2;
 
+  pyqrcode = buildPythonPackage rec {
```

Comment:
We are moving Python expressions out of `pkgs/top-level/python-packages.nix` into `pkgs/development/python-modules/<module>/default.nix`. 

Please move the expression there, and call it from `pkgs/top-level/python-packages.nix` using `callPackage ../development/python-modules/<package> { };`.